src/System/Console/Hawk/Runtime/HaskellExpr.hs

```diff
@@ -4,13 +4,12 @@ module System.Console.Hawk.Runtime.HaskellExpr where
 
 import qualified Data.ByteString.Lazy.Char8 as B
 
-import Data.HaskellExpr
-import System.Console.Hawk.Representable
-import System.Console.Hawk.Runtime
-import System.Console.Hawk.Runtime.Base
+import           Data.HaskellExpr
+import           System.Console.Hawk.Runtime
+import           System.Console.Hawk.Runtime.Base
 
 
-eSomeRows :: Rows a => HaskellExpr (a -> SomeRows)
+eSomeRows :: HaskellExpr (a -> SomeRows)
```

Comment:
I understand why this causes a warning, but the constraint needs to stay. Each `HaskellExpr` contains a string representing a Haskell expression, and is indexed by a phantom type indicating the type of the expression. Here, the type of the `System.Console.Hawk.Runtime.SomeRows` constructor (runtime/System/Console/Hawk/Runtime.hs#20) is `Rows a => a -> SomeRows`, and so the constraint is important.